### PR TITLE
Update to use OpenStudio 3.5.0 rc1

### PR DESCRIPTION
**Pull request by tijcolem**
A few changes in here to support 3.5.0-rc1

- Windows was failing due to have "' '" wrapped around the osw_path, which is odd considering this worked fine for previos versions so something has changed and might need to updated on SDK side. 
- Minor updates to the algo .zip files for 3.5.0 changes 
- Have to disable urbanopt as the common-measures gem includes an older version of standards (0.2.16) which breaks on SDK 3.5.0 due to some deprecated methods that have been removed. We may have to wait until all the measures are all released before including it.

### CODE REVIEW

File: server/app/jobs/dj_jobs/run_simulate_data_point.rb [hunk 1]
```diff
@@ -208,7 +208,7 @@ def perform
               uo_process_log = File.join(simulation_dir, 'urbanopt_process.log')
               run_urbanopt(uo_simulation_log, uo_process_log)
             else  #OS CLI workflow
-              cmd = "#{Utility::Oss.oscli_cmd(@sim_logger)} #{@data_point.analysis.cli_verbose} run --workflow '#{osw_path}' #{@data_point.analysis.cli_debug}"
+              cmd = "#{Utility::Oss.oscli_cmd(@sim_logger)} #{@data_point.analysis.cli_verbose} run --workflow #{osw_path} #{@data_point.analysis.cli_debug}"
```
Comment (tijcolem) [inline server/app/jobs/dj_jobs/run_simulate_data_point.rb:211]: @brianlball This is the change I'm not sure about. This was the simulation error log on windows. Somehow extra quotes are being passed. This change does work but haven't tested this with spaces in path names. 

```
I, [2022-10-25T16:10:12.930845 #436]  INFO -- : CLI Parsed Inputs: ["--verbose"] "run" ["--workflow", "'C:/projects/openstudio-server/spec/files/temp_data/analysis_5ac83f5d-e867-4047-91ce-859505fd2d18/data_point_5235afa3-d680-4509-af82-f4308df9b700/data_point.osw'", "--debug"]
D, [2022-10-25T16:10:12.930949 #436] DEBUG -- : Main arguments are ["--verbose"]
D, [2022-10-25T16:10:12.930973 #436] DEBUG -- : Sub-command is run
D, [2022-10-25T16:10:12.931011 #436] DEBUG -- : Sub-arguments are ["--workflow", "'C:/projects/openstudio-server/spec/files/temp_data/analysis_5ac83f5d-e867-4047-91ce-859505fd2d18/data_point_5235afa3-d680-4509-af82-f4308df9b700/data_point.osw'", "--debug"]
...
...
Error executing argv: ["--verbose", "run", "--workflow", "'C:/projects/openstudio-server/spec/files/temp_data/analysis_5ac83f5d-e867-4047-91ce-859505fd2d18/data_point_5235afa3-d680-4509-af82-f4308df9b700/data_point.osw'", "--debug"]
Error: Invalid argument @ realpath_rec - C:/projects/openstudio-server/server/'C: in eval:510:in `realpath'
eval:510:in `realpath'
:/ruby/2.7.0/gems/openstudio-workflow-2.3.1/lib/openstudio/workflow/run.rb:141:in `realpath'
```

Comment (brianlball) [reply server/app/jobs/dj_jobs/run_simulate_data_point.rb:211]: @tijcolem yeah, looks like something changed and it doesnt like the single quotes.  Those have been in there for years.

Comment (tijcolem) [reply server/app/jobs/dj_jobs/run_simulate_data_point.rb:211]: Yeah, I can do some more testing on it with 3.5.0-rc1 along with SDK if we need to revert back and wrap single quotes.

**Comment by tijcolem** [comment]
The failing CI runner is due to an older version in git in measure-tester-gem. 
see: https://github.com/NREL/OpenStudio-measure-tester-gem/blob/v0.3.1/openstudio_measure_tester.gemspec#L32 
@brianlball Looks like you updated this but we'll need a patched release 0.3.2. We can release that but it won't get updated until the 3.5.0-rc2 release.

**Review by brianlball — APPROVED** [review]

**Pull request merged**